### ISSUE TITLE
Canvas-height is 0 when image height is bigger than image width (Firefox)

### ISSUE THREAD

**Issue by lpsBetty** [alexk111/ngImgCrop#71]
When I upload an image in Firefox the generated canvas is not visible.. This happens only when the uploaded image is in portrait format (height bigger than width),

the generated canvas:
````
<canvas width="100" height="0" style="margin-top: 0px; margin-left: -50px; cursor: default;"></canvas>
`````
Can someone confirm this?

**Comment by fernoftheandes** [comment]
No, if you load a portrait image into the fiddle I posted for issue #46, it works without problems. I am using Firefox 37.0.1.

**Comment by lpsBetty** [comment]
hmm doesn't work for me with Firefox 37.0.2 on Mac with image size 2592 × 4608

**Comment by lpsBetty** [comment]
can someone confirm this?
generated canvas from http://jsfiddle.net/L2tae1to/ using Firefox 38.0.1 on Mac:
````
<canvas style="margin-top: 0px; margin-left: -50px; cursor: default;" height="0" width="100"></canvas>
````

**Comment by xudao520** [comment]
having the same issue，Did you solve this?

**Comment by xudao520** [comment]
@lpsBetty Did you solve this?

**Comment by lpsBetty** [comment]
@xudao520 I tried my fiddle again, and it seems that it works now, can you confirm?

http://jsfiddle.net/L2tae1to/

**Comment by xudao520** [comment]
@lpsBetty 
I forgot to tell you that I used for testing device is mobile， 
And I just did a test with pc good, but there are still some tests with iphone5s image in question

**Comment by dongseok0** [comment]
I got same problem on iPhone6.
